browser-test/src/admin_program_creation.test.ts

```diff
@@ -76,4 +78,64 @@ describe('Create program with enumerator and repeated questions', () => {
 
     await endSession(browser)
   })
+
+  fit('change questions order within block', async () => {
```

Comment:
```suggestion
  it('change questions order within block', async () => {
```

Reply:
Thanks for catching it. We should enable eslint check for that.